server/routes/reporting/standard-report/standard-report-print.js

```diff
@@ -305,7 +305,7 @@ async function standardReportPrint(app, req, res, reportKey, data) {
 };
 
 function sortTableData({ sortBy, sortDirection }, tableData, reportData) {
-  if (reportData.bespokeReport) return; // we do not sort bespoke reports for now
+  if (reportData.bespokeReport.body) return; // we do not sort bespoke reports for now
```

Comment:
CHange to the following?
`if (reportData.bespokeReport?.body) return;`

Breaks when report definitions dont have bespokeReport